tests/integration/devfile/cmd_devfile_delete_test.go

```diff
@@ -17,11 +17,13 @@ var _ = Describe("odo devfile delete command tests", func() {
 	const devfile = "devfile.yaml"
 	var devfilePath string
 	var componentName, invalidNamespace string
+	var oc helper.OcRunner
 
 	var commonVar helper.CommonVar
 
 	// This is run before every Spec (It)
 	var _ = BeforeEach(func() {
+		oc = helper.NewOcRunner("oc")
```

Comment:
The  `oc` runner won't work on kubernetes clusters. Please use the `commonVar.CliRunner` and introduce a new function which doesn't wait for the resource deletion to be completed.

Reply:
Updated